packages/webpack/src/index.ts

```diff
@@ -118,7 +118,7 @@ export default function WebpackPlugin<Theme extends object>(
                   : (result.getLayer(layer) || '')
 
                 if (!quote)
-                  return css
+                  return css.replaceAll(/\\/g, '\\\\\\\\')
```

Comment:
Why do we need so many slashes?

Reply:
I'm not sure but I'm not good with webpack. I found this with trial and error. So if somebody have better webpack-fu, please advice. Having said that, dumping this `RawSource`:

https://github.com/unocss/unocss/blob/5d0a295860edc6c905af831b29ef9050f73086ff/packages/webpack/src/index.ts#L131

We can look for the output which is something like this:
```js
eval(
  /*v this is one single string to be eval-ed,  
       v with inlined backtick */
    "<... webpack content ...>
     ___CSS_LOADER_EXPORT___.push([
       module.id,
       `<... unocss' css in backtick ...>`,
       \"\"]);
    <... more webpack content ...>")
```
I hope you can see where this is going.

The css in the backtick (after this PR) already be outputted as 4x `\`:
![image](https://github.com/unocss/unocss/assets/379924/795b27de-9c82-45dc-be48-f55ce3e7644d)

And my guess is within the 2 quotes, the outer string ("") and the inner literal (\`\`), all the backslashes were reduced down to single `\` for output.

I think more escaping needs to be done with the quoted branch, but let's wait for issue or reproduction.
https://github.com/unocss/unocss/blob/5d0a295860edc6c905af831b29ef9050f73086ff/packages/webpack/src/index.ts#L120-L128